modules/browser/BrowserCapabilities.js

```diff
@@ -139,17 +128,15 @@ export default class BrowserCapabilities extends BrowserDetection {
     supportsBandwidthStatistics() {
         // FIXME bandwidth stats are currently not implemented for FF on our
         // side, but not sure if not possible ?
-        return !this.isFirefox() && !this.isEdge()
-            && !this.isSafariWithWebrtc();
+        return !this.isFirefox() && !this.isSafariWithWebrtc();
     }
 
     /**
      * Checks if the current browser supports WebRTC datachannels.
      * @return {boolean}
      */
     supportsDataChannels() {
-        // NOTE: Edge does not yet implement DataChannel.
-        return !this.isEdge();
+        return true;
```

Comment:
Maybe let's dump this function altogether?

Reply:
sure